### PR TITLE
Added Bulk Send to Vivial Connect SDK

### PR DESCRIPTION
**Pull request by magils**

### CODE REVIEW

File: src/test/java/net/vivialconnect/tests/MessageTest.java [hunk 1]
```diff
@@ -102,6 +106,63 @@ public void test_redact_message() throws VivialConnectException {
...
+        for(AssociatedNumber number: numbers.subList(1, numbers.size())){
+            bulkSendNumberList.add(number.getPhoneNumber());
+        }
+
+        Message message = new Message();
+
+        message.setFromNumber(fromNumber);
+        message.setBody("Testing Bulk send...");
+        message.setToNumbers(bulkSendNumberList);
+        message.setToNumbers(bulkSendNumberList);
```
Comment (warchiefx) [inline src/test/java/net/vivialconnect/tests/MessageTest.java:125]: Why setting this twice?

File: src/test/java/net/vivialconnect/tests/MessageTest.java [hunk 2]
```diff
@@ -102,6 +106,63 @@ public void test_redact_message() throws VivialConnectException {
...
+
+        BulkInfo bulkInfo = getDataSource().sendBulk(message);
```
Comment (warchiefx) [inline src/test/java/net/vivialconnect/tests/MessageTest.java:127]: Shouldn't this be `message.sendBulk()`?

File: src/main/java/net/vivialconnect/model/message/Message.java [hunk 3]
```diff
@@ -136,6 +144,12 @@
      */
     private List<String> mediaUrls;
 
+    /**
+     * ID of the bulk to which this message belongs.
+     */
+    @JsonProperty("bulk_id")
+    private String bulkId;
```
Comment (warchiefx) [inline src/main/java/net/vivialconnect/model/message/Message.java:151]: I'd prefer the Bulk API to be a separate object than re-using the Message class. Should be easier to understand. Thoughts?

**Review by warchiefx — CHANGES_REQUESTED** [review]

**Comment by magils** [comment]
These feebacks were addressed in a new branch and new pull request
https://github.com/Localvox/vivialconnect-java/pull/7

**Pull request closed without merging**